tests/unit/test_objective_plot_data.py

```diff
@@ -1,45 +1,101 @@
 import os
 import pandas as pd
+import numpy as np
 import pytest
 
 import everviz
 
-from everviz.pages.objectives import _calc_p10_p90, _set_up_data_sources, _calc_mean
+from everviz.pages.objectives import (
+    _set_up_data_sources,
+    _objective_values,
+    _objective_statistics,
+    _total_objective_values,
+)
 
 OBJECTIVES = [
-    {"batch": 0, "realization": 1, "function": "f0", "value": 100, "simulation": None},
-    {"batch": 0, "realization": 1, "function": "f1", "value": 200, "simulation": None},
-    {"batch": 2, "realization": 2, "function": "f0", "value": 200, "simulation": None},
-    {"batch": 2, "realization": 2, "function": "f1", "value": 400, "simulation": None},
-    {"batch": 0, "realization": 3, "function": "f0", "value": 300, "simulation": None},
-    {"batch": 0, "realization": 3, "function": "f1", "value": 600, "simulation": None},
-    {"batch": 2, "realization": 4, "function": "f0", "value": 400, "simulation": None},
-    {"batch": 2, "realization": 4, "function": "f1", "value": 800, "simulation": None},
+    {"batch": 0, "realization": 1, "function": "f0", "value": 100, "simulation": 0},
+    {"batch": 0, "realization": 1, "function": "f1", "value": 200, "simulation": 1},
+    {"batch": 2, "realization": 2, "function": "f0", "value": 200, "simulation": 2},
+    {"batch": 2, "realization": 2, "function": "f1", "value": 400, "simulation": 3},
+    {"batch": 0, "realization": 3, "function": "f0", "value": 300, "simulation": 4},
+    {"batch": 0, "realization": 3, "function": "f1", "value": 600, "simulation": 5},
+    {"batch": 2, "realization": 4, "function": "f0", "value": 400, "simulation": 6},
+    {"batch": 2, "realization": 4, "function": "f1", "value": 800, "simulation": 7},
 ]
 
 
+def test_objective_values_data_frame():
+    """Test for the correct layout and size of the objective values data frame."""
+    objective_values = _objective_values(pd.DataFrame(OBJECTIVES))
+
+    assert set(
+        objective_values.columns
+        == set(["batch", "function", "realization", "simulation"])
+    )
```

Comment:
Not really sure what this is testing... but the result of
```
set(
        objective_values.columns
        == set(["batch", "function", "realization", "simulation"])
    )
```
is {False} which evaluates to `True`, this seems a bit strange to me 

did you want to check this?
```
assert set(objective_values.columns) == {"batch", "function", "value", "simulation"}
```

Reply:
Indeed, good catch!